README.md

```diff
@@ -1,17 +1,20 @@
-# corrp  <a href='http://meantrix.com'><img src='man/figures/logo.png' align="right" height="139" /></a>
-
+# corrp  
+<a href='http://meantrix.com'><img src='man/figures/logo.png' align="right" height="139" /></a>
 <!-- badges: start -->
 
-[![version](https://img.shields.io/badge/version-0.2.0-green.svg)](https://semver.org)
+[![version](https://img.shields.io/badge/version-0.3.0-green.svg)](https://semver.org)
 [![License: GPL3](https://img.shields.io/badge/License-GPL3-green.svg)](https://www.gnu.org/licenses/gpl-3.0.en.html)
 
 <!-- badges: end -->
 
-Compute multiple types of correlations analysis (Pearson correlation, R^2  coefficient of linear regression, Cramer's V measure of association, Distance Correlation,The Maximal Information Coefficient, Uncertainty coefficient and Predictive Power Score) in large dataframes with mixed columns classes(integer, numeric, factor and character) in parallel backend.
+Compute multiple types of correlations analysis (Pearson correlation, R^2  coefficient of linear regression, Cramer's V measure of association, Distance Correlation,The Maximal Information Coefficient, Uncertainty coefficient and Predictive Power Score) in large dataframes with mixed columns classes(integer, numeric, factor and character) in parallel R backend.
```

Comment:
...of correlation analysis... (remove "s")
...Distance Correlation, The Maximal... **(space missing after comma)**
...classes (integer... **(space missing before parenthesis)**